src/Adyen/Service/Checkout.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Adyen\Service;
+
+
+class Checkout extends \Adyen\Service
+{
+    protected $_setup;
+    protected $_verify;
+    protected $_paymentMethods;
+    protected $_payments;
+    protected $_paymentsDetails;
+
+    public function __construct(\Adyen\Client $client)
+    {
+        parent::__construct($client);
+
+        $this->_setup = new \Adyen\Service\ResourceModel\Checkout\Setup($this);
+        $this->_verify = new \Adyen\Service\ResourceModel\Checkout\Verify($this);
+        $this->_paymentMethods = new \Adyen\Service\ResourceModel\Checkout\PaymentMethods($this);
+        $this->_payments = new \Adyen\Service\ResourceModel\Checkout\Payments($this);
+        $this->_paymentsDetails = new \Adyen\Service\ResourceModel\Checkout\PaymentsDetails($this);
+        $this->_supportsXAPIKey = true;
```

Comment:
now that the pal is accepting apiKey, we should better change it to "requiresApiKey".
Preferably bring another \Adyen\Service subclass (eg. ApiKeyAuthenticatedService) that has it set by default and let Checkout extending the new service class